README.md

```diff
@@ -1,3 +1,15 @@
 # The DMV
 
 This is the starter repo for the BE Mod1 DMV project.
+Reflections:  
+# Q1-- Digging into code base. What was your process?
+
+Q1 Answer-- First I examined the lib files individually, side by side with it's respective spec file. Without touching the lib file code, I examined what the tests required, and wrote down on paper what I thought would need to be added or changed. Close file, rinse and repeat with the next file, until I had a working tree for reference of each file before actually testing. Then I run the test files and compare with my answers.
```

Comment:
Great job using the test files as a guide. That's exactly what they are, documentation of how the code should behave.